espnet3/systems/tts/f5_tts/inference.py

```
@@ -0,0 +1,559 @@
+                "'reference_speech' (cross/same-speaker) or 'speech' (self-reference)."
+            )
+
+        if isinstance(text, (list, tuple)):
+            reference_text = (
+                reference_text if reference_text is not None else [None] * len(text)
+            )
+            wavs = [
+                self.infer_one(one_text, one_reference_audio, one_reference_text)
+                for one_text, one_reference_audio, one_reference_text in zip(
+                    text, reference_audio, reference_text
+                )
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)